xbmc/cores/AudioEngine/Utils/AEBitstreamPacker.cpp

```diff
@@ -193,11 +230,11 @@ void CAEBitstreamPacker::PackTrueHD(CAEStreamInfo &info, uint8_t* data, int size
     }
   }
 
-  for (nextCodeIdx = 0; nextCodeIdx < ARRAY_SIZE(MatCodes); nextCodeIdx++)
+  for (nextCodeIdx = 0; nextCodeIdx < static_cast<int>(MatCodes.size()); nextCodeIdx++)
```

Comment:
Header `Util.h` can be removed now as only was used for `ARRAY_SIZE` macro

Reply:
Thanks! Updated.